### DISCUSSION TITLE
Missing documentation on multi-copier project updates

### DISCUSSION THREAD

**Discussion by hwine** [copier-org/copier#334] [Q&A]
One of the [touted features](https://www.pythonpodcast.com/copier-project-scaffolding-episode-297/) of copier is the ability to apply multiple templates to the same target project.

However, there are at least two "tricky spots":
1. How to initially apply the templates when there could be multiple which used the default answer file name?
2. How to apply updates from various templates? Especially those which have a conflict on the answer file name.

My initial attempts to figure this out have failed. Pointers wanted!

**Comment by yajo** [comment]
The answer to both questions is to specify the answers file, using `-a`.

https://copier.readthedocs.io/en/stable/configuring/#answers_file

**Reply by hwine** [reply]
My experience was that didn't work as advertised. Now knowing that was the correct approach, I'll put together STR and open a bug.

Those docs are useful for folks who write templates, but the information isn't presented in a way that is simple for a "pure user" to understand.  Are you open to a "quick start for end user" section in the docs?

For example, I don't see a "native" way to determine the default answer file location prior to install. So the user has to understand enough about template writing to examine the source repository prior to installation, and determine the default.

**Reply by yajo** [reply]
> Are you open to a "quick start for end user" section in the docs?

Of course! I value a lot docs experience from newbies, and i believe good docs is essential! 



> For example, I don't see a "native" way to determine the default answer file location prior to install. So the user has to understand enough about template writing to examine the source repository prior to installation, and determine the default.

Are you missing something in copier then, or in the docs? 🤔
